### PR TITLE
[BugFix][Quantization] Restrict norm-quant fusion to INT8

### PR DESCRIPTION
**Pull request by Oseltamivir**
### What this PR does / why we need it?

Constrain both INT8 norm-quant fusion patterns with `dst_type=torch.int8`. Otherwise, FP8 quantization can match an INT8-only replacement and fail during the following FP8 matmul.

CI compatibility: restore optional xlite mask access (refs #16658).

### Does this PR introduce _any_ user-facing change?

FP8 models can start with norm-quant fusion enabled. Implicit and explicit INT8 calls still fuse.

### How was this patch tested?

- Ascend 950PR: four FP8 regression cases fail before the fix; 12 cases pass afterward. Twelve bias cases skip because the custom operator is unavailable.
- Llama 3.3 70B FP8 starts and passes eight answer smoke checks.
- Five isolated xlite cache-routing cases pass.
- `bash format.sh ci` passes.

Hardware tests used equivalent current-main pattern classes in an installed v0.26 overlay; the full current-main engine was not tested.

Developed with OpenAI Codex assistance.

- vLLM main: https://github.com/vllm-project/vllm/commit/84030bbe3d74d99bad477a3d2e37a973ccd8865c

### CODE REVIEW

File: tests/e2e/pull_request/one_card/compile/test_dynamic_quant_fusion_dtype.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+from vllm_ascend.compilation.passes.norm_quant_fusion_pass import (
+    AddRMSNormDynamicQuantPattern,
+    AddRMSNormDynamicQuantPatternWithBias,
+)
+from vllm_ascend.utils import is_950
+
+try:
+    import npugraph_ex as nge
+except ImportError:
+    import torchair as nge
```
[the remark this answers is not in the document]
Comment (Oseltamivir) [reply tests/e2e/pull_request/one_card/compile/test_dynamic_quant_fusion_dtype.py:24]: Addressed in [01ebd1c](https://github.com/Oseltamivir/vllm-ascend/commit/01ebd1cba626bb33dfe399e1f4617d58efd88755). The test now obtains the backend callable from the imported package using `get_npu_backend(compiler_config=...)`: `npugraph_ex` mode for NPUGraph-EX and `reduce-overhead` mode for the TorchAir fallback, matching the production compiler setup. This avoids assuming that a backend named `torchair` is registered.

Validated package selection for NPUGraph-EX only, TorchAir only, and both available using mocked compiler packages with real CPU `torch.compile`. Formatting and mypy for Python 3.10/3.11/3.12 pass. These are backend-selection checks; a fresh NPU run still requires the maintainer test label.

File: tests/e2e/pull_request/one_card/compile/test_dynamic_quant_fusion_dtype.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    def inspect_graph(gm, *args, **kwargs):
+        result = original_optimize(gm, *args, **kwargs)
+        optimized_targets.extend(node.target for node in gm.graph.nodes if node.op == "call_function")
+        return result
+
+    torch._dynamo.reset()
+    with torch.no_grad(), patch.object(nge.npu_fx_compiler, "_optimize_fx", inspect_graph):
+        expected = model(x, residual, weight, bias)
+        actual = torch.compile(model, backend="npugraph_ex", fullgraph=True, dynamic=True)(x, residual, weight, bias)
```
[the remark this answers is not in the document]
Comment (Oseltamivir) [reply tests/e2e/pull_request/one_card/compile/test_dynamic_quant_fusion_dtype.py]: Addressed in [01ebd1c](https://github.com/Oseltamivir/vllm-ascend/commit/01ebd1cba626bb33dfe399e1f4617d58efd88755): `torch.compile` now receives the callable returned by the selected compiler package instead of a hardcoded registry name. The three backend-selection smoke checks and Python 3.10/3.11/3.12 type checks pass.
